client/src/components/Search/style.css

```diff
@@ -0,0 +1,110 @@
+.search__form {
+  display: flex;
+  flex-direction: column;
+  max-width: 98%;
+  padding: 1rem;
+  margin: auto;
+}
+
+.place-name {
+  text-align: left;
+  width: 100%;
+  margin: auto;
+  margin-bottom: 1rem;
+}
+
+.place-name--label {
+  margin: 0;
+}
+
+.country-city-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.autocomplete-box {
+  text-align:left;
+  width:49%;
+}
+
+.type-filter__container {
+  display: flex;
+  justify-content: space-between;
+  margin: 1rem 0;
+}
+
+.radio-group {
+  max-width: 70%;
+  margin-left: 0.25rem;
+}
+
+.button-label {
+  text-align: left;
+  margin: 0;
+}
+
+.form-action {
+  display: flex;
+  justify-content: space-around;
+}
+
+.country-dropdown {
+  margin-right: 1rem;
+}
+
+.radio-group .ant-radio-button-wrapper {
+  margin-right: 0.5rem !important;
+}
+
+.ant-radio-button-wrapper:hover {
+  background-color: var(--main-color);
+  opacity: 0.9;
+  color: #fff;
+}
+
+.ant-radio-button-wrapper:last-child {
+  border-radius: 0 !important;
+}
+
+.ant-radio-group > .ant-radio-button-wrapper-checked {
+  border: none !important;
+  background-color: var(--main-color) !important;
+  color: #fff !important;
+  font-weight: bold !important;
+}
+
+.keywords-btn {
+  border-radius: 0;
+}
+
+.search-btn {
+  background-color: var(--main-color) !important;
+  color: #fff !important;
+  font-size: 1.25rem !important;
+  border: none;
+  width: 34%;
+  border-radius: 0;;
+}
+
+.search-btn:hover {
+  background-color: var(--main-color);
+  color: #fff;
+  border: none;
+}
+
+.recommendation-btn {
+  background-color: var(--main-color) !important;
+  color: #fff !important;
+  border: none;
+  width: 65%;
+  opacity: var(--opacity-search-button);
+  border-radius: 0;
+}
+
+.type-filter__container__filter {
+  text-align: left;
+}
+
+.ant-select-arrow-icon {
+  color: var(--main-color) !important;
+}
```

Comment:
the class names are inconsistent. Some use BEM correctly, some incorrectly and some don't follow it at all. I'm not gonna ask you to fix it as it's not the most important thing in the world and thus not worth the trouble, but try to keep it in mind and be consistent in the future

Reply:
also, there is a lot of `!important` rules in the CSS, which doesn't seem right. Most of the time they can and should be avoided
https://css-tricks.com/when-using-important-is-the-right-choice/

Reply:
this is an ant design class we must use `!important` to our CSS effect to an element

Reply:
It should be possible to override ant's CSS with your styles without `!important` (not sure how much work that would be). However there are other classes defined here (`.search-btn`, `recommendation-btn`) that also use `!important` that looks suspicious. But most importantly, which I didn't realize before - why are you overriding global styles defined by ant in CSS file for the Search component? This file should only contain styles defined for the component! If you wanna override ant's default styles you should do it in the top-level CSS module.

Reply:
yes, thank you so much for these useful comments 😃